.dockerignore

```diff
@@ -3,7 +3,6 @@
 **/.dockerignore
 **/Dockerfile
 **/.gitignore
-**/.git
```

Comment:
The main downside of this is that it will invalidate the `node-builder` cache with any changes to it. I'm up to merge this to fix the immediate issue, but we might want to consider passing this info into the docker build args in the make command instead of the current system to reduce that issue

Reply:
refactored to optionally use environment variables